packages/rpc-methods/src/restricted/getBip32Entropy.test.ts

```diff
@@ -89,7 +20,7 @@ describe('validateCaveatPaths', () => {
           type: SnapCaveatType.PermittedDerivationPaths,
           value,
         }),
-      ).toThrow('Expected non-empty array of paths.');
+      ).toThrow(EthereumRpcError);
```

Comment:
I'd prefer testing for error text, in this case using a RegEx or error prefix, in case we unexpectedly change error messages in the future.

Reply:
There are different errors thrown for different values, I'd have to separate the tests to test separately for `undefined`, `null`, `object`, etc.

Reply:
This should work:
```suggestion
      ).toThrow(/Error prefix: .+\./u);
```